analytics.js

```diff
@@ -280,12 +235,8 @@ class Analytics {
       }
 
       // determine the dataPlaneUrl
-      try {
-        this.serverUrl = this.resolveDataPlaneUrl(response.source.dataPlaneUrls);
-      } catch (e) {
-        handleError(e);
-        this.serverUrl = this.serverUrl ? this.serverUrl : DEFAULT_DATAPLANE_URL;
-      }
+      this.serverUrl = resolveDataPlaneUrl(response);
```

Comment:
`serverUrl` is not sent here.

Reply:
Also, `options`.

Reply:
Thank you for pointing out, addressed it